src/middlewared/middlewared/plugins/account.py

```diff
@@ -868,7 +857,14 @@ def recreate_homedir_if_not_exists(self, has_home, user, group, mode):
                     'options': {'stripacl': True},
                 }).wait_sync(raise_error=True)
 
-    @accepts(Int('id'), Dict('options', Bool('delete_group', default=True)))
+    @accepts(
+        Int('id'),
+        Dict(
+            'options',
+            Bool('delete_group', default=True),
+            Bool('delete_ssh', default=True),
```

Comment:
This doesn't need to be an option. When a user is deleted, we should always remove the .ssh dir.